docs/Compliance.md

```diff
@@ -0,0 +1,124 @@
+# ATO and compliance considerations
+
+Every federal information system must be granted an Authority To Operate (ATO)
+by an Authorizing Official in order to go into production.  A good overview of
+the process can be found in this [excellent ATO overview](https://docs.google.com/presentation/d/1x-Bt8uyW-szHarglY57fFcC6PRCpUGFLe07uOvt93uk/edit#slide=id.g1f710bd7ce_0_847)
+and a more in-depth look can be found in [18F's before-you-ship docs](https://before-you-ship.18f.gov/ato/).  There
+are currently a few options for getting an ATO:
+* A [FedRAMP Tailored](https://tailored.fedramp.gov/) package.
+* A full [FedRAMP JAB ATO](https://www.fedramp.gov/jab-authorization/).
+* A GSA LATO (Lightweight ATO), which can be found by searching for "Lightweight
```

Comment:
A GSA LATO would not be applicable- please remove

Reply:
Done!